test/permissions.spec.ts

```diff
@@ -46,6 +46,12 @@ describe.skip(options.WEBKIT)('permissions', () => {
     expect(await getPermission(page, 'geolocation')).toBe('granted');
   });
 
+  it('should not grant geolocation permission when listed', async({page, server, context}) => {
```

Comment:
nit: the name is confusing, should it be something like 'should prompt geolocation permission when not listed'?

Reply:
Absolutely!